bandit.conf

```diff
@@ -85,7 +85,7 @@
 tests:
 
 # (optional) list skipped test IDs here, eg '[B101, B406]':
-# skips: ['B603', 'B607', 'B404', "B608"]
+skips: ['B603', 'B607', 'B404', "B608"]
```

Comment:
Do we actually need to disable B608?  I can see that we'd need to do the subprocess ones but B608 is sqlite-related and I don't think anything in this repo talks to the database directly?